hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/DirectoryDeletingService.java

```diff
@@ -265,7 +287,11 @@ private boolean previousSnapshotHasDir(
           String prevDbKey = prevDirTableDBKey == null ?
               metadataManager.getOzoneDeletePathDirKey(key) : prevDirTableDBKey;
           OmDirectoryInfo prevDirInfo = prevDirTable.get(prevDbKey);
-          return prevDirInfo != null &&
+          //Check the snapshot chain hasn't changed while the checking the previous snapshot.
```

Comment:
```suggestion
          // Check the snapshot chain hasn't changed while checking the previous snapshot.
```
Can you please add what we are checking in the previous snapshot?

Reply:
done